BLaurenB/techtime
# Rspec and helpers - updated and cleaned

What's this PR do?

Builds on the gem branch, added rspec and spec folders, as well as added required gems and data cleaning to the spec/rails_helper file

Where should the reviewer start- what are the pertinent changes and the sequence?

Added rspec from the command line (rails generate rspec:install)
Added the following to the rails_helper:
require ‘rspec/rails'
require ‘spec_helper'
require 'capybara/rails'
require ‘database_cleaner’
DatabaseCleaner.strategy = :truncation
Configure Factory Bot, factory_bot_rails
config.include FactoryGirl::Syntax::Methods
config.before(:each) do
DatabaseCleaner.clean
end
config.after(:each) do
DatabaseCleaner.clean
end
How should this be manually tested?

I'm not sure. we write some tests and see if we get any gem-related errors?

Any background context you want to provide?

Just trying to get things set up for TDD!!

What are the relevant tickets?

I created Waffle ticket #5

Screenshots (if appropriate)

n/a

Questions:

Just let me know if we should incoporate into Dev and Master branches.
